src/main/java/duke/duke/commands/Command.java

```diff
@@ -0,0 +1,35 @@
+package duke.commands;
+
+import duke.info.exception.InvalidInputException;
+import duke.info.task.Calendar;
+import duke.ui.Ui;
+import duke.storage.Storage;
+
+public abstract class Command {
+
+    /**
+     * Constructs an empty Command
+     */
+    Command() {}
```

Comment:
Should it all be on a single line?
```suggestion
    Command() {
    }
```